### ISSUE TITLE
[RFC]: add support for `prefers-reduced-motion` media query to disable animations

### ISSUE THREAD

**Issue by kgryte** [stdlib-js/www#25]
### Description

This RFC proposes adding support for `prefers-reduced-motion` media query to disable animations. Animations can cause physical discomfort for some people, and we should respect user preferences in this regard.

In particular, this applies to the API docs side menu where we use animation for sliding out the drawer and for collapsing/expanding namespaces.

### Related Issues

No.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

**Comment by Planeshifter** [comment]
Implemented per c0198595aa6f636eba7a70eaae54c2b72ecbb021.

**Comment by kgryte** [comment]
@Planeshifter Can we go ahead and close this? Or is there more work to be done?

**Comment by Planeshifter** [comment]
Yes! Should be done.
